### PR TITLE
SPCTR-2399 - Form block and form's child removed unwanted codes

### PR DESCRIPTION
**Pull request by rahulvarma722**
### Description
Remove unwanted codes from 14 blocks.
### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by rajmohanreddy774 — APPROVED** [review]

**Review by AbhijitNage123 — APPROVED** [review]

File: src/blocks/forms/child-blocks/accept/render.js [hunk 1]
```diff
@@ -18,9 +18,7 @@ const Render = ( props ) => {
 	const { block_id, acceptRequired, acceptText, showLink, linkLabel, link, linkInNewTab } = attributes;
 
 	const isRequired = acceptRequired ? __( 'required', 'ultimate-addons-for-gutenberg' ) : '';
```
Comment (supriyajjadhav) [inline src/blocks/forms/child-blocks/accept/render.js:20]: We can remove the translation from here also.

File: src/blocks/forms/child-blocks/accept/save.js [hunk 2]
```diff
@@ -12,9 +12,7 @@ export default function save( props ) {
 	const { block_id, acceptRequired, acceptText, showLink, linkLabel, link, linkInNewTab } = attributes;
 
 	const isRequired = acceptRequired ? __( 'required', 'ultimate-addons-for-gutenberg' ) : '';
```
Comment (supriyajjadhav) [inline src/blocks/forms/child-blocks/accept/save.js:14]: We can remove the translation from here also.

File: src/blocks/forms/child-blocks/upload/edit.js [hunk 3]
```diff
@@ -1,23 +1,15 @@
 /**
  * BLOCK: Forms - Upload - Edit
```
Comment (supriyajjadhav) [inline src/blocks/forms/child-blocks/upload/edit.js:2]: @rahulvarma722  We are not currently using the upload child block in our forms. We plan to develop a new block to replace it.
Let's delete the upload folder and please make sure that there are no dependencies on the block.
cc: @RajkiranBagal

**Review by supriyajjadhav — APPROVED** [review]

**Comment by adi3890** [comment]
I could see lot of strings getting changed from translatable to non-translatable should we consider for rare cases where if strings were translated and may cause block recovery issue.
Just a concern.
@rahulvarma722 @RajkiranBagal

**Comment by rahulvarma722** [comment]
> I could see lot of strings getting changed from translatable to non-translatable should we consider for rare cases where if strings were translated and may cause block recovery issue. Just a concern. @rahulvarma722 @RajkiranBagal

Deprecated file added

**Review by adi3890 — APPROVED** [review]

**Pull request merged**